storage/Utils/AppUtil.h

```diff
@@ -106,6 +108,31 @@ void classic(StreamType& stream)
     string sformat(const string& format, ...);
 
 
+    /**
+     * Pads the string s to at least width w with char c at the front.
+     */
+    string
+    pad_front(const string& s, size_t w, char c);
+
+
+    struct NameSchema
+    {
+	NameSchema(regex re, size_t w, char c) : re(re), w(w), c(c) {}
+
+	const regex re;
```

Comment:
Can I ask for more meaningful name for the fields of this struct. `re`, `w` and `c` are not the quintessence of clarity. :wink: